From: Jeremy Bicha <jeremy.bicha@canonical.com>
Date: Tue, 15 Aug 2023 20:22:35 -0400
Subject: Disable non working camera/microphones panels

Upstream: https://gitlab.gnome.org/GNOME/gnome-control-center/issues/381
https://gitlab.gnome.org/GNOME/gnome-control-center/-/issues/741
https://gitlab.gnome.org/GNOME/cheese/-/issues/95

Even with improved text from the original implementation, the panel
doesn't seem to work with Snap apps. I disabled the camera and microphone
here and the Zoom snap still works and it isn't listed as an app
"requesting permission".
---
 panels/privacy/cc-privacy-panel.ui | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/panels/privacy/cc-privacy-panel.ui b/panels/privacy/cc-privacy-panel.ui
index fdfcdb5..9820070 100644
--- a/panels/privacy/cc-privacy-panel.ui
+++ b/panels/privacy/cc-privacy-panel.ui
@@ -93,6 +93,7 @@
                           <object class="CcListRow">
                             <property name="title" translatable="yes">_Cameras</property>
                             <property name="subtitle" translatable="yes">Control camera access</property>
+                            <property name="visible">False</property>
                             <property name="icon-name">camera-symbolic</property>
                             <property name="use-underline">True</property>
                             <property name="show-arrow">True</property>
@@ -105,6 +106,7 @@
                           <object class="CcListRow">
                             <property name="title" translatable="yes">_Microphones</property>
                             <property name="subtitle" translatable="yes">Control microphone access</property>
+                            <property name="visible">False</property>
                             <property name="icon-name">microphone-symbolic</property>
                             <property name="use-underline">True</property>
                             <property name="show-arrow">True</property>
